apps/rod/sections/summary-data-sections.js

```diff
@@ -60,6 +60,26 @@ module.exports = {
     {
       step: '/main-applicant',
       field: 'main-applicant-nationality'
+    },
+    {
+      step: '/reference-number',
+      field: 'enter-reference-number'
```

Comment:
field name is different from the pages.json entry for label. It should be `reference-number`.

And if more than one option is selected, the value needs to be parsed and should display in the summary page as like below.

```
Case ID
Home Office reference number
Payment reference number
Courier reference number
```